.appveyor.yml

```diff
@@ -0,0 +1,39 @@
+# -*- mode: yaml -*-
+
+environment:
+
+  matrix:
+    - CONFIG: win_c_compilervs2015python3.6
+      CONDA_INSTALL_LOCN: C:\Miniconda36-x64
+
+    - CONFIG: win_c_compilervs2015python3.7
+      CONDA_INSTALL_LOCN: C:\Miniconda36-x64
+
+
+# We always use a 64-bit machine, but can build x86 distributions
+# with the TARGET_ARCH variable.
+platform:
+    - x64
+
+install:
+    # If there is a newer build queued for the same PR, cancel this one.
+
+    # Cygwin's git breaks conda-build. (See https://github.com/conda-forge/conda-smithy-feedstock/pull/2.)
+    - cmd: rmdir C:\cygwin /s /q
```

Comment:
Might be quicker to rename it (`rename c:\cygwin c:\cygwin-dont-use-me`)

Reply:
right. I must confess I just copied over pieces from the slycot-feedstock autogenerated yml